pyproject.toml

```diff
@@ -23,17 +23,18 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
+python = "^3.6.2"
 importlib-metadata = { version = ">1.5.1", python = "<3.8" }
```

Comment:
do we know why we restrict python to be <3.8? This is not as ideal as allowing up to current major release

Reply:
i'm fairly certain that in poetry this:
```
importlib-metadata = { version = ">1.5.1", python = "<3.8" }
```

means 'use a version of  importlib-metadata higher than 1.5.1 when a python version less than 3.8 is used'

Reply:
ah and then just use latest version otherwise?  learned something new. that's cool